pages/record/_.vue

```diff
@@ -1,52 +1,64 @@
 <template>
-  <section class="container">
-    <h1 class="title">
-      Record
-    </h1>
-
-    <p v-if="error">
-      <strong>Error:</strong> {{ error }}
-    </p>
-    <template v-else>
-      <div>
+  <p v-if="error">
```

Comment:
The error message is now outside of the container, all the way over to the left. Is that intentional?

Reply:
No, so I made some changes.